### PR TITLE
Feat: amendments and revisions page

### PR DESCRIPTION
**Pull request by mikevespi**
[ZH Card](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/919)

### CODE REVIEW

**Review by pbastia — CHANGES_REQUESTED** [review]
A minor refactor, we might want to do this in a separate PR to speed things up

**Comment by BCerki** [comment]
There's a bug where the table sizing bounces around after filtering: https://www.loom.com/share/9a293b7088cb42219005003e43a262ce

**Review by bc-cas — DISMISSED** [review]
looks great!! just needs a rebase

**Comment by Sepehr-Sobhani** [comment]
> There's a bug where the table sizing bounces around after filtering: https://www.loom.com/share/9a293b7088cb42219005003e43a262ce

Thanks for noticing this. going to make a tech debt card for that and merge this PR to unblock other issues.

File: app/cypress/integration/cif/project-revision-change-logs.spec.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    cy.get("h2").contains(/Amendments & Other Revisions/i);
+    cy.happoAndAxe("Amendments & Other Revisions", "view", "main");
+    cy.get("tbody tr").should("have.length", 5);
+    cy.get("[placeholder='Filter']").first().type("Minor");
+    cy.get("button").contains("Apply").click();
+    cy.get("tbody tr").should("have.length", 1);
+    cy.get("button").contains("Clear").click();
+    cy.get("tbody tr").should("have.length", 5);
+    cy.get("thead th").contains("Effective Date").click();
+    cy.get("thead th").contains("Effective Date").click();
```
[the remark this answers is not in the document]
Comment (Sepehr-Sobhani) [reply app/cypress/integration/cif/project-revision-change-logs.spec.js:28]: It's not a duplicate, we need to click this field twice to get the DESC order but you're right, this is confusing so I'm gonna add a comment here for future reference.

**Review by pbastia — APPROVED** [review]

**Pull request merged**